korangar/src/graphics/passes/point_shadow/shader/entity.wgsl

```diff
@@ -119,6 +136,50 @@ fn vertex_data(vertex_index: u32) -> Vertex {
     return Vertex(vec3<f32>(x, y, z), vec2<f32>(u, v), depth, curve);
 }
 
+fn vertex_data_new(vertex_index: u32, instance_index: u32) -> Vertex {
```

Comment:
Dito as in the bindless point shader file. I won't repeat this for the other shaders.